build_image.sh

```diff
@@ -15,14 +15,14 @@ fi
 setup_type=$1
 app_path=$PWD
 app_name=${app_path##*/}
-parent_path=`dirname $PWD`
+#parent_path=`dirname $PWD`
 
 echo "-- Generating image for $setup_type" container.
 
 mkdir -p $app_path/workspace/docker_certificates
 
 echo "-- Copying host config file to be used by docker in current directory"
-cp $parent_path/config/cchost_config_${setup_type}_js.json $app_path/workspace/docker_certificates/.
+cp $app_path/config/cchost_config_${setup_type}_js.json $app_path/workspace/docker_certificates/.
```

Comment:
The point of putting the config files in the sample was so that we didn't have to copy files around. I think it would make more sense to change the dockerfile

Old line:
COPY ./workspace/docker_certificates/cchost_config_virtual_js.json /app/
To be:
COPY ./config/cchost_config_virtual_js.json /app/

When you've done this, you don't need the copy line in this script

(Remember to also update the enclave docker file)

Reply:
Acknowleged!

Reply:
Done!